includes/abstracts/abstract-batch.php

```diff
@@ -73,13 +88,60 @@
 	public $total_num_results;
 
 	/**
+	 * Get results function for the registered batch process.
+	 *
+	 * @return array
+	 */
+	abstract public function individual_get_results();
```

Comment:
what does the `individual_` naming convention represent? An individual batch? Maybe `batch_` would be a better prefix in that case? 🌴